Library/Homebrew/env_config.rb

```diff
@@ -11,6 +11,11 @@ module EnvConfig
     module_function
 
     ENVS = {
+      HOMEBREW_ALLOWED_TAPS:                     {
+        description: "A space-separated list of taps. Homebrew will refuse to install a " \
```

Comment:
May be worth documenting what happens if a tap is both allowed and forbidden? Also: that official taps are always allowed.

Reply:
That official taps are always allowed is documented in this env variable.

Adding a tap to both the allowed and forbidden list will forbid it. The error message will show explicitly that it is because the tap is listed in `HOMEBREW_FORBIDDEN_TAPS`. I think that suffices in place of making the documentation more verbose.